File: CTDK/070/070.cpp
#include <iostream>
using namespace std;

int main()
{
	float x;
	cout << "Nhap x: ";
	cin >> x;

	int n;
	cout << "Nhap n: ";
	cin >> n;

	float s = 0;
	float t = x * x;
	int i = 1;
	while (i <= n)
	{
		t = t * x * x;
		s = s + t;
		i = i + 1;
	}
	cout << "Tong la: " << s << endl;
	return 0;
}

File: CTDK_Python/070/_070.py
x = float(input("Nhap x: "))
n = int(input("Nhap n: "))

s = 0
t = 1
i = 2
while(i <= 2 * n):
    t = t * x * x
    s = s + t
    i = i + 2

print("Tong la: ", s)
